src/newcharacter.cpp

```diff
@@ -567,7 +567,7 @@ bool avatar::create( character_type type, const std::string &tempname )
     // Learn recipes
     for( const auto &e : recipe_dict ) {
         const auto &r = e.second;
-        if( !r.has_flag( "SECRET" ) && !knows_recipe( &r ) && has_recipe_requirements( r ) ) {
+        if( !knows_recipe( &r ) && has_recipe_requirements( r ) ) {
```

Comment:
This seems wrong.

Reply:
Yeah, some garbage caused by my computer crashing, corrupting .git/HEAD, and then missing something in cleanup.